command-expansion-server/src/app.ts

```diff
@@ -37,7 +37,7 @@ const PORT: number = parseInt(getEnv().PORT, 10) ?? 27184;
 const app: Application = express();
 // WARNING: bodyParser.json() is vulnerable to a string too long issue. Iff that occurs,
 // we should switch to a stream parser like https://www.npmjs.com/package/stream-json
```

Comment:
Do we feel like we understand this comment? "vulnerable to a string too long issue"? What risks (if any) are there associated with raising the limit?

Reply:
No idea. I think it's low risk because we will just get an `entity to large` error if we try to send a string that is too large. @dyst5422 can probably elaborate.

Reply:
The limit is essentially there to avoid abusive payloads. A stream parser is going to be significant processing-time overhead, so unless we find ourselves in a situation that legitimately needs it, it's best to avoid. The hard limit on string length is 256mb, so we are well below that with the 100mb limit.

Reply:
Thanks, both of you :+1:

Reply:
And yes, @camargo is right that it's not a server-breaking issue if we attempt to load pass something larger.

Reply:
We hit the string too long issue after ~1.5 years of operations on M2020 in a case where we were returning all the simulation results from SSIM (a very large payload) and did the switch to stream-json at that point, which slows things down for smaller payloads, but was necessary there, so there's prior art on lab in doing that shift.